solana-programs/anchor/audius-data/cli/main.ts

```diff
@@ -0,0 +1,288 @@
+import { Program, Provider, Wallet, web3 } from "@project-serum/anchor";
+import ethWeb3 from "web3";
+import { Connection, PublicKey, Keypair } from "@solana/web3.js";
+import { AudiusData } from "../target/types/audius_data";
+import * as anchor from "@project-serum/anchor";
+const { SystemProgram, Transaction, Secp256k1Program } = anchor.web3;
+import {
+  getTransaction,
+  getRandomPrivateKey,
+  ethAddressToArray,
+  randomCID,
+  findDerivedPair,
+} from "../lib/utils";
+import {
+  initUserSolPubkeyArgs,
+  initAdmin,
+  initUser,
+  initUserSolPubkey,
+  createTrack,
+  createTrackArgs,
+} from "../lib/lib";
+
+import { Command } from "commander";
+const program = new Command();
+
+const EthWeb3 = new ethWeb3();
+
+const idl = JSON.parse(
+  require("fs").readFileSync("./target/idl/audius_data.json", "utf8")
+);
+
+const opts: web3.ConfirmOptions = {
+  skipPreflight: true,
+  preflightCommitment: "confirmed",
+};
+
+const keypairFromFilePath = (path: string) => {
+  return Keypair.fromSecretKey(Uint8Array.from(require(path)));
+};
+
+/// Initialize constants requird for any CLI functionality
+function initializeCLI(ownerKeypairPath: string) {
+  // const network = "https://solana-api.projectserum.com";
+  const network = "https://audius.rpcpool.com/4d12c27ad978e40c1b0f8449b93c";
```

Comment:
should we pass the network in as an argument to the function, and maybe have a default?

Reply:
yeah this is a good call - will add